js/hotgraphicView.js

```diff
@@ -282,6 +285,8 @@ define([
         },
 
         preRemove: function() {
+            if (!this.completionEvent) return;
```

Comment:
completionEvent will always be set as far as I can tell, it'll default to "allItems" if it's not specified:

https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/1c7b004d960dd006879cd9105a69a3c172bd39b9/js/hotgraphicView.js#L257